### DISCUSSION TITLE
Invalid argument while issuing a kernel command

### DISCUSSION THREAD

**Discussion by LukasAfonso** [ipxe/ipxe#1067] [General]
I am loading the following script via the command `chain` however it gives me an error that the argument is invalid with the error code 1c346002.
I pretty much copied the documentation so I don't know what is wrong here. I enabled NFS via `echo "#define DOWNLOAD_PROTO_NFS" >> config/local/general.h`.

```
set server_ip 192.168.1.201
set nfs_path /pxe/template
kernel nfs://${server_ip}${nfs_path}/vmlinuz
initrd nfs://${server_ip}${nfs_path}/initrd.img
imgargs vmlinuz initrd=initrd.img root=/dev/nfs boot=casper netboot=nfs nfsroot=${server_ip}:${nfs_path} ip=dhcp splash quiet --
boot
```

**Comment by NiKiZe** [comment]
So the kernel command gives you an error message? What is that full error message, including the url to ipxe.org? And what does that page recommend to test?

**Reply by LukasAfonso** [reply]
I have put the error code into the question so the error link is ipxe.org/1c346002. And it says invalid argument as stated by the question.

**Reply by NiKiZe** [reply]
Yes post the link so that anyone trying to help you don't have to generate the link on their own.

One other thing (among several) that the page says is "Try building iPXE with the [debug](https://ipxe.org/download#debug_builds) option DEBUG=nfs_open" it will probably give you limited help, but might give you hints.

**Reply by LukasAfonso** [reply]
Hum, weirdly enough it does not tell me any errors besides the one I mentioned.
Here is a screenshot of me doing it manually:
![image](https://github.com/ipxe/ipxe/assets/101110269/86b68835-9b95-494e-bd49-25f0c84cf1c5)
